pytket/extensions/qujax/qujax_convert.py

```diff
@@ -231,8 +234,16 @@ def tk_to_qujax(
         or Callable[[jnp.ndarray = None], jnp.ndarray]
         if no parameters found in circuit
     """
+    qujax_args = tk_to_qujax_args(circuit, symbol_map)
 
-    return qujax.get_params_to_statetensor_func(*tk_to_qujax_args(circuit, symbol_map))
+    if simulator == "statetensor":
```

Comment:
Have you thought about doing an enum or an bool for this value?

Reply:
how would an enum look?

I thought about bool `(densitytensor=False)` but concluded this was clearer